src/snowflake/snowpark/modin/pandas/series_utils.py

```diff
@@ -187,46 +183,46 @@ def split(
             - If found splits <= n, make all splits
             - If for a certain row the number of found splits < n, append None for padding up to n if expand=True
             - If using expand=True, Series and Index callers return DataFrame and MultiIndex objects, respectively.
-
-        Examples
-        --------
-        >>> s = pd.Series(
-            [
-                "this is a regular sentence",
-                "https://docs.python.org/3/tutorial/index.html",
-                np.nan
-            ]
-        )
-        s
-        0                       this is a regular sentence
-        1    https://docs.python.org/3/tutorial/index.html
-        2                                              NaN
-        dtype: object
-
-        In the default setting, the string is split by whitespace.
-
-        >>> s.str.split()
-        0                   [this, is, a, regular, sentence]
-        1    [https://docs.python.org/3/tutorial/index.html]
-        2                                                NaN
-        dtype: object
-
-        The n parameter can be used to limit the number of splits on the delimiter.
-
-        >>> s.str.split(n=2)
-        0                     [this, is, a regular sentence]
-        1    [https://docs.python.org/3/tutorial/index.html]
-        2                                                NaN
-        dtype: object
-
-        The pat parameter can be used to split by other characters.
-
-        >>> s.str.split(pat="/")
-        0                         [this is a regular sentence]
-        1    [https:, , docs.python.org, 3, tutorial, index...
-        2                                                  NaN
-        dtype: object
         """
+        # TODO: SNOW-1432416 Add examples to docstring when implemented.
```

Comment:
This API is now raising NotImplementedError even though it was previously implemented. I didn't want to get rid of the doctests so I commented it out.